sdk/spring/azure-spring-cloud-stream-binder-servicebus-queue/README.md

```diff
@@ -106,6 +107,16 @@ Set it to `true` if a message that trigger any exception in consumer will be for
 Set it to `false` if a message that trigger any exception in consumer will be re-queued. 
 
 Default: `false`
+
+**_receiveMode_**
+
+The modes for receiving messages.
+
+`PEEK_LOCK`, received message is not deleted from the queue or subscription, instead it is temporarily locked to the receiver, making it invisible to other receivers.
+
+`RECEIVE_AND_DELETE`, received message is removed from the queue or subscription and immediately deleted.
+
+Default: `PEEK_LOCK`
```

Comment:
Should we add introdution for maxConcurrentCalls as well?

Reply:
It already has an introduction to maxConcurrentCalls [here](https://github.com/zhichengliu12581/azure-sdk-for-java/blob/dc8756197ea3c09fec4caa9215933105bc5eebab/sdk/spring/azure-spring-cloud-stream-binder-servicebus-queue/README.md#L94)